packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx

```diff
@@ -81,7 +81,7 @@ function CardCategory({item}: {item: PropSidebarItemCategory}): JSX.Element {
         {count: item.items.length},
       )}
     />
-  );
+  ) : null;
```

Comment:
same as before, I'd like any comp in DocCard to never render null as it's the responsibility of the list to filter or not items upfront

Reply:
That's because the `findFirstCategoryLink` function can return `undefined`. But since we have a validation rule that prevents empty items in categories, we can assume that there will always be a link.